okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/IoRemoteException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2017 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting3.okhttp;
+
+import java.io.IOException;
+
+// TODO(rfink): Can we hide this as an implementation detail of RemotingOkHttpCall?
```

Comment:
I think this wrapper class warrants some Javadoc justifying the subtle reasons we had to implement it.

Reply:
done. also changed the wrapped type.